.env.example

```diff
@@ -58,4 +58,7 @@ TRANSACTIONS_RETRY_OFFSET=60
 CLOSURE_RETRY_INTERVAL=30
 
 JWT_SECRET=ODMzNUZBNTZENDg3NTYyREUyNDhGNDdCRUZDNzI3NDMzMzQwNTFEREZGQ0MyQzA5Mjc1RjY2NTQ1NDk5MDMxNzU5NDc0NUVFMTdDMDhGNzk4Q0Q3RENFMEJBODE1NURDREExNEY2Mzk4QzFEMTU0NTExNjUyMEExMzMwMTdDMDk
-JWT_TOKEN_VALIDITY_TIME_MILLIS=600000
\ No newline at end of file
+JWT_TOKEN_VALIDITY_TIME_MILLIS=600000
+
+CONFIDENTIAL_DATA_MANAGER_KEY=EWsfdHUIDwLwYwC0zIMlrw==
```

Comment:
I don't like the fact that this env variable is referring to an implementation detail, what about `ENCRYPTION_SECRET_KEY`? Maybe it's a bit too generic...

Reply:
I don't know, maybe `EMAIL_ENCRYPTION_KEY` is a better choice just to empathize that this encryption key is used only for mail encryption and that, for future confidential data a new key must be used rather that this. What do you think?

Reply:
Fixed into https://github.com/pagopa/pagopa-ecommerce-transactions-service/pull/150/commits/c0ff765bde5bc8d25a69ff0d7e92d77dae01ef6f